Open/pages/index.vue

```diff
@@ -109,27 +109,58 @@ const openModule = namespace("open");
     components: {
         OpenButton,
     },
+    // head () {
```

Comment:
whats with this commented out head block

Reply:
Was trying to see if I could do something with the i18n seo